packages/@adobe/spectrum-css-temp/components/popover/index.css

```diff
@@ -48,7 +48,6 @@ governing permissions and limitations under the License.
 
   outline: none; /* Hide focus outline */
   box-sizing: border-box;
-  overflow: hidden;
```

Comment:
This needs to be there for other popovers. Perhaps we can override it specifically for menus?

Reply:
Just merged this into https://github.com/adobe/react-spectrum/pull/4917 and added this back. Also setting it back to visible for menus.